packages/cloud_firestore/android/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-firestore:[11.6.0,12.0['
-        api 'com.google.firebase:firebase-core:[11.6.0,12.0['
+        api 'com.google.firebase:firebase-firestore:15.+'
+        api 'com.google.firebase:firebase-core:15.+'
```

Comment:
In cases where we are using libraries other than firebase-core we can exclude the firebase-core dependency.

Reply:
Done.